File: VB/DocumentPrintPreview/Employee.vb
Imports System.Drawing
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI

Namespace DocumentPrintPreview

    Public Partial Class EmployeeReport
        Inherits DevExpress.XtraReports.UI.XtraReport

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Overrides Sub CreateDocument(ByVal buildForInstantPreview As Boolean)
            MyBase.CreateDocument(buildForInstantPreview)
        End Sub
    End Class
End Namespace

File: CS/DocumentPrintPreview/Employee.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;

namespace DocumentPrintPreview
{
	public partial class EmployeeReport : DevExpress.XtraReports.UI.XtraReport
	{	
		public EmployeeReport()
		{
			InitializeComponent();
		}

        public override void CreateDocument(bool buildForInstantPreview) {
            base.CreateDocument(buildForInstantPreview);
        }

    }
}
